xill-processor/src/test/resources/tests/web/DownloadHTTPSFromNotTrusted.xill

```diff
@@ -22,7 +22,7 @@
 use Web;
 use Assert;
 
-var testURL = "https://bugs.xillio.nl";
+var testURL = "https://www.xillio.com/hubfs/css-includes/logo-xillio-white.png";
```

Comment:
I would personally not pick this resource for a download test as it might change in the future. I would rather go with one of those test data sets that haven't been changed in the last 20 years. I believe a government of some country has one of these sites. Thomas might know more.